concepts/fundamentals/introduction.md

```diff
@@ -0,0 +1,138 @@
+# Introduction
+
+We'll refer to the _language_ as **AWK** and to the _executable_ that interprets AWK programs as **`awk`** or **`gawk`**.
```

Comment:
* Do you want to touch on how people actually use/write awk on the terminal? eg [this slide](https://docs.google.com/presentation/d/1l9S_6QB8ES5vrp573Ms578qhSEC0xE_xjazJKNyHoSI/preview?slide=id.p10) and/or [this slide](https://docs.google.com/presentation/d/1l9S_6QB8ES5vrp573Ms578qhSEC0xE_xjazJKNyHoSI/preview?slide=id.p18)
* Do you want to touch on user defined vars?

Reply:
I added the first slide. The others can wait for subsequent concepts.